### PR TITLE
DatePicker Ux Updates

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#6803

*Description of changes:*

* Minor UX updates.

* Make calendar icon clickable and opens datepicker.

* Override inherited opacity.

### CODE REVIEW

File: src/components/DatePicker/DatePicker.vue [hunk 1]
```diff
@@ -26,6 +26,7 @@
                       v-on="on"
                       v-on:keydown="$event.preventDefault()"
                       v-on:keyup.enter="emitDate(dateText)"
+                      v-on:click:append="on.click"
```
Comment (severinbeauvais) [inline src/components/DatePicker/DatePicker.vue:29]: We might want to add this to our other date pickers... I suggest a new ticket.

**Review by severinbeauvais — APPROVED** [review]

File: src/components/DatePicker/DatePicker.vue (hunk 1 above)
Comment (cameron-freshworks) [reply src/components/DatePicker/DatePicker.vue:29]: For sure! What other Date Pickers do we have? In the create UI maybe?

Comment (severinbeauvais) [reply src/components/DatePicker/DatePicker.vue:29]: Off the top of my head:
- FED in Create UI
- AR Date in Filings UI

Comment (cameron-freshworks) [reply src/components/DatePicker/DatePicker.vue:29]: Okie doke will do!

Comment (cameron-freshworks) [reply src/components/DatePicker/DatePicker.vue:29]: Ticket here :) : https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/7017

Comment (severinbeauvais) [reply src/components/DatePicker/DatePicker.vue:29]: Perfect, thanks!

**Pull request merged**